SmartDeviceLink/SDLNames.h

```diff
@@ -321,13 +356,25 @@ extern SDLName const SDLNamePrimaryAudioSource;
 extern SDLName const SDLNamePRNDL;
 extern SDLName const SDLNameProgressIndicator;
 extern SDLName const SDLNameProperties;
+extern SDLName const SDLNamePS;
```

Comment:
Change these RDS two letter ones into the full out text for clarity, though the value will remain the same.